deploy/host_vars/staging.yml

```diff
@@ -57,15 +57,6 @@ k8s_auth_api_key: !vault |
   39663030626532623063633033373236373835323266633235326563313132386139643535336563
   363132353764303637656564323632636334
 
-database_password: !vault |
-  $ANSIBLE_VAULT;1.1;AES256
-  31633236353234623634616635306235633431373936393632656266333831333335326565353536
-  6565353434336561386234346639626634363164613139620a333134643231643634373137323638
-  62366363386436613935323163663562313865326236306662333765613565353037386332323134
-  3631346233636465350a646135386165336631383837383436653638346361333862663035613066
-  61383363666334303639353661353862363433383833643164623865636363383162343666636132
-  3135306661623561633630323062613065623738383866653833
```

Comment:
This value should be preserved and moved to `staging_shared.yml`, but I see a different encrypted password there